tools/cldr-code/src/test/java/org/unicode/cldr/unittest/TestPersonNameFormatter.java

```diff
@@ -158,5 +167,30 @@ public void TestLabels() {
         return result;
     }
 
+    private void assertThrows(String subject, Runable code) {
+        try {
+            code.run();
+            fail(String.format("%s was supposed to throw an exception.", subject));
+        }
+        catch (Exception e) {
+            assertTrue("Exception was thrown as expected.", true);
+        }
+    }
+
+    private void assertDoesNotThrow(String subject, Runable code) {
+        try {
+            code.run();
+            assertTrue("Exception was not thrown as expected.", true);
+        }
+        catch (Exception e) {
+            fail(String.format("%s was not supposed to throw an exception.", subject));
+        }
+    }
+
+    @FunctionalInterface
+    private interface Runable {
```

Comment:
The correct spelling is `Runnable`, but I'm pretty sure Java already defines a "Runnable" interface.  Although it looks a little bit like the built-in Java `Runnable` interface might be what you want here...

Reply:
Oh, thanks, that's exactly what I wanted!